app/src/main/java/org/p2p/wallet/striga/exchange/models/StrigaExchangeRate.kt

```diff
@@ -0,0 +1,12 @@
+package org.p2p.wallet.striga.exchange.models
+
+import java.math.BigDecimal
+import org.p2p.core.utils.MillisSinceEpoch
+
+data class StrigaExchangeRate(
+    val price: BigDecimal,
+    val buyRate: BigDecimal,
+    val sellRate: BigDecimal,
+    val timestamp: MillisSinceEpoch,
+    val currency: String,
```

Comment:
currencyUnits?

Reply:
it's a currency name like: `Euros`, `Tether`, `USD Coin`; so I'll rename it to `currencyName`